banjo/conf/schema.xml

```diff
@@ -205,7 +205,10 @@
    <field name="acknowledgement_party" type="string" indexed="true" stored="true" multiValued="true"/>
    <field name="acknowledgement_type" type="string" indexed="true" stored="true" multiValued="true"/>
    <field name="work_created_during_migration" type="boolean" indexed="true" stored="true"/>
-   
+   <field name="additional_series_statement" type="text_general" indexed="true" stored="true"/>
+   <field name="sheet_name" type="text_general" indexed="true" stored="true"/>
+   <field name="sheet_creation_date" type="int" indexed="true" stored="true"/>

```

Comment:
just followed existing example, expiry_date, start_date, end_date, all in the type of int. I don't think there is such type as date in schema.